sass/components/_barrier.scss

```diff
@@ -123,26 +125,54 @@
 /*hover*/
 
 .price {
+    display: block;
     font-size: 12px;
     font-weight: bold;
-    display: block;
-    padding: 3px 1px;
     line-height: 18px;
-    overflow: hidden;
+    overflow: unset;
+    padding: 3px 1px;
+    position: relative;
+    right: 45px;
     text-align: center;
-}
+    width: max-content;
```

Comment:
Cool) I never used this style before:)

Reply:
This is one of the most useful things in CSS for me)